### DISCUSSION TITLE
8Jan24 Update

### DISCUSSION THREAD

**Discussion by jasonkwm** [Bunyod16/42-on_campus_stats#54] [Show and tell]
Looks good with the recent dashboard update
![20240108_165356.jpg](https://github.com/Bunyod16/42-on_campus_stats/assets/32697686/18c76d77-752e-40cd-b1d2-2a68cde4c5bc)

**Comment by jasonkwm** [comment]
@Jachokoreto @sirhcofe @Bunyod16

**Reply by Jachokoreto** [reply]
Broken image lol but yeah its Aaaammaaazzziinnggff
